lv_objx/lv_chart.c

```diff
@@ -821,4 +886,248 @@ static void lv_chart_draw_vertical_lines(lv_obj_t * chart, const lv_area_t * mas
         }
     }
 }
+
+static void lv_chart_draw_y_ticks(lv_obj_t * chart, const lv_area_t * mask)
+{
+    lv_chart_ext_t * ext = lv_obj_get_ext_attr(chart);
+
+    if(ext->y_axis.list_of_values != NULL || ext->y_axis.num_tick_marks != 0) {
+
+        const lv_style_t * style = lv_obj_get_style(chart);
+        lv_opa_t opa_scale = lv_obj_get_opa_scale(chart);
+
+        uint8_t i,j;
+        uint8_t list_index;
+        uint8_t num_of_labels;
+        uint8_t num_scale_ticks;
+        uint8_t major_tick_len, minor_tick_len;
+        lv_point_t p1;
+        lv_point_t p2;
+        lv_coord_t x_ofs = chart->coords.x1;
+        lv_coord_t y_ofs = chart->coords.y1;
+        lv_coord_t h = lv_obj_get_height(chart);
+        lv_coord_t w = lv_obj_get_width(chart);
+        char buf[16+1]; /* up to 16 symbols per label + null terminator */
```

Comment:
Please make `16` into a define at the top of the file instead of using magic numbers. This also applies to the `x_ticks` variant.